user-guide/Troubleshooting/toc.yml

```diff
@@ -205,6 +205,11 @@ items:
         items:
           - name: Elasticsearch/OpenSearch does not start up after upgrade or disk cleanup
             topicUid: Elastic_not_starting_after_upgrade_cleanup
+      - name: Troubleshooting - SLScripting
```

Comment:
Added to TOC but [Troubleshooting_procedures](https://github.com/SkylineCommunications/dataminer-docs/pull/3644/files#diff-4813ed67d08f1831573f73cc7bf8d41edd62e618c792c748d5a73844daafc146R184) is also a page with a list that could probably be adjusted.

Reply:
updated